src/db_init.h

```diff
@@ -26,7 +26,7 @@
  * is a major upgrade. In other words minor version upgrades permit downgrading
  * forked-daapd after the database was upgraded. */
 #define SCHEMA_VERSION_MAJOR 21
-#define SCHEMA_VERSION_MINOR 04
+#define SCHEMA_VERSION_MINOR 05
```

Comment:
Won't downgrading and running with the new database mess up the persistent id's?